docs/Modifying_Clearwater_settings.md

```diff
@@ -31,7 +31,7 @@ The following settings in `/etc/clearwater/shared_config` can safely be changed
 To change one of these settings, if you are using Clearwater's [automatic configuration sharing](Automatic_Clustering_Config_Sharing) functionality:
 
 *   Edit `/etc/clearwater/shared_config` on *one* node and change to the new value.
-*   Run `/usr/share/clearwater/bin/upload_shared_config` to upload the new config to etcd.
+*   Run `/usr/share/clearwater/clearwater-config-manager/scripts/upload_shared_config` to upload the new config to etcd.
 *   On each node in turn run `TODO: AMC`. This will download the new config to the node and restart the necessary services to pick up the new settings.
```

Comment:
* Run `/usr/share/clearwater/clearwater-config-manager/scripts/apply_shared_config` to upload the new config to etcd.  This will...